pkg/catalog/install.go

```diff
@@ -19,15 +19,17 @@ type Clients struct {
 
 // ProfileConfig contains configuration for profiles ie. catalogName, profilesName, etc.
 type ProfileConfig struct {
-	CatalogName   string
-	ConfigMap     string
-	Namespace     string
-	ProfileName   string
-	SubName       string
-	Version       string
-	ProfileBranch string
-	URL           string
-	Path          string
+	GitRepoName      string
```

Comment:
this is getting rather large. Would you mind groupping them?

GitRepoNamespace,GitRepoName -> GitRepoDetails
ProfileName,ProfileBranch,URL(ProfileURL),SubName(ProfileSubscriptionName?) -> ProfileDetails

For a better readability.

Reply:
sounds good!

Reply:
done!